portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,50 @@
 package com.google.sps.servlets;
 
+import java.io.BufferedReader;
 import java.io.IOException;
+import java.io.PrintWriter;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+/**
+ * Handles requests sent to the /hello URL. Try running a server and navigating
+ * to /hello!
+ */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
```

Comment:
Since the body of this class is not "Hello World!" any more, you should change its name to what it does.

Reply:
Yes, I will change the name

Reply:
Actually since it's part of the feature that displays on the screen hello "name_input_from the user" hello might still be a good name.

Reply:
Yes I think "Hello" can be a good name, but not "HelloWorld" for class name :)